3001-4000/LIT3165Menas.xml

```diff
@@ -76,6 +76,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <ptr target="bm:Budge1909Mena"/>
                   <citedRange>44-58</citedRange>
                </bibl>
+               <bibl><ptr target="bm:1910KaufmannMena"/></bibl>
```

Comment:
Thank you for adding the new record to Zotero, please make sure that you **always** follow Zotero guidelines and provide full information exactly as on the frontispiece, including the **publisher** AND the **series** 

The citation should not include any names that are not on the frontispiece


![image](https://github.com/user-attachments/assets/50708072-ac7d-43fc-b2c4-37257b0a0344)

Reply:
I have tried!